src/playwright-runner.js

```diff
@@ -264,6 +264,7 @@ async function run (nodeBin, runCfgPath, suiteName) {
 
   process.env.PLAYWRIGHT_CFG_FILE = runCfg.playwright.configFile || '';
   process.env.BROWSER_NAME = suite.param.browserName;
+  process.env.HEADLESS = !suite.param.headed || false;
```

Comment:
I'd recommend sticking to the `headless` term. Neither of those places lists `headed` as an attribute:
- https://playwright.dev/docs/debug#run-in-headed-mode
- https://playwright.dev/docs/ci#running-headed
- https://playwright.dev/docs/api/class-testoptions#test-options-headless

```suggestion
  process.env.HEADLESS = suite.param.headless;
```

Reply:
ok! I'll update to `headless`. I was also confused why there are two config fields `headed` and `headless`.